Classes/KIFSystemTestActor.m

```diff
@@ -58,7 +58,7 @@ - (void)simulateMemoryWarning
 
 - (void)simulateDeviceRotationToOrientation:(UIDeviceOrientation)orientation
 {
-    [[UIDevice currentDevice] setValue:[NSNumber numberWithInt:orientation] forKey:@"orientation"];
+    [[UIDevice currentDevice] setValue:[NSNumber numberWithInt:(int)orientation] forKey:@"orientation"];
```

Comment:
Probably could just use `@(orientation)` and avoid the explicit cast. That'd be cleaner if it works IMO.

Reply:
Tested it locally and it fixes the compiler error. Went ahead and made that change in this branch.

Reply:
Nice, definitely cleaner. Thanks for the fast follow up!